app/src/renderer/components/transactions/LiStakeTransaction.vue

```diff
@@ -58,9 +58,7 @@
         Undelegation
         <b>
           {{
-            full(
-              calculatePrettifiedTokens(tx.validator_address, tx.shares_amount)
-            )
+            calculatePrettifiedTokens(tx.validator_address, tx.shares_amount)
```

Comment:
does `calculatePrettifiedTokens` include `full` ?

Reply:
Yes, that's why it includes "prettified"